## 05_html_docs_css.patch by Y Giridhar Appaji Nag <giridhar@appaji.net>
##
## Remove google trackers and adjust css to look like online splint docs

--- a/doc/bugs.html
+++ b/doc/bugs.html
@@ -622,7 +622,7 @@ href="http://www.splint.org/bugs.html">R
 Bugs</a> -
 <a
 href="http://www.splint.org/lists.html">Mailing
-Lists</a> &nbsp;&nbsp;<IMG src="http://sourceforge.net/sflogo.php?group_id=9538" width="0" height="0" alt="">
+Lists</a> 
 
 &nbsp;&nbsp;
 <a
@@ -640,16 +640,6 @@ href="http://www.splint.org/credits.html
 </tr>
 </table>
 
-<script type="text/javascript">
-var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
-document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
-</script>
-<script type="text/javascript">
-var pageTracker = _gat._getTracker("UA-3775212-3");
-pageTracker._initData();
-pageTracker._trackPageview();
-</script>
-
 </body>
 </html>
 
--- a/doc/changes.html
+++ b/doc/changes.html
@@ -763,7 +763,7 @@ href="http://www.splint.org/bugs.html">R
 Bugs</a> -
 <a
 href="http://www.splint.org/lists.html">Mailing
-Lists</a> &nbsp;&nbsp;<IMG src="http://sourceforge.net/sflogo.php?group_id=9538" width="0" height="0" alt="">
+Lists</a> 
 
 &nbsp;&nbsp;
 <a
@@ -781,16 +781,6 @@ href="http://www.splint.org/credits.html
 </tr>
 </table>
 
-<script type="text/javascript">
-var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
-document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
-</script>
-<script type="text/javascript">
-var pageTracker = _gat._getTracker("UA-3775212-3");
-pageTracker._initData();
-pageTracker._trackPageview();
-</script>
-
 </body>
 </html>
 
--- a/doc/html/faq.html
+++ b/doc/html/faq.html
@@ -4,7 +4,7 @@
   <head>
     <meta name="generator"
     content="HTML Tidy for Linux/x86 (vers 1st November 2002), see www.w3.org" />
-    <link rel="stylesheet" type="text/css" href="http://www.splint.org/splint.css"
+    <link rel="stylesheet" type="text/css" href="splint.css"
     title="style1" />
     <title>Splint FAQ</title>
   </head>
@@ -561,7 +561,7 @@ href="http://www.splint.org/bugs.html">R
 Bugs</a> -
 <a
 href="http://www.splint.org/lists.html">Mailing
-Lists</a> &nbsp;&nbsp;<IMG src="http://sourceforge.net/sflogo.php?group_id=9538" width="0" height="0" alt="">
+Lists</a> 
 
 &nbsp;&nbsp;
 <a
@@ -579,16 +579,6 @@ href="http://www.splint.org/credits.html
 </tr>
 </table>
 
-<script type="text/javascript">
-var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
-document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
-</script>
-<script type="text/javascript">
-var pageTracker = _gat._getTracker("UA-3775212-3");
-pageTracker._initData();
-pageTracker._trackPageview();
-</script>
-
 </body>
 </html>
 
--- a/doc/html/manual.css
+++ b/doc/html/manual.css
@@ -10,8 +10,8 @@ title { font-family: Ariel, helvetica, s
 A:link { color: blue }
 A:visited { color: darkblue }
 A:active { color: green }
-body { margin-left: 0%; margin-right: 0%; margin-top: 0%; margin-bottom:
-0% }
+body { margin-left: 1%; margin-right: 1%; margin-top: 1%; margin-bottom:
+1% }
 h4 { margin-left: 0%; margin-right: 0% }
 h3 { margin-left: 0%; margin-right: 0% }
 h2 { margin-left: 0%; margin-right: 0% }
--- a/doc/html/manual.htm
+++ b/doc/html/manual.htm
@@ -4,7 +4,7 @@
 "HTML Tidy for Solaris (vers 1st March 2003), see www.w3.org"
       name="generator">
 <link rel="stylesheet" type="text/css" href=
-"../manual.css" title="style1">
+"manual.css" title="style1">
 <title>Splint Manual</title>
 <style type="text/css">
 <!--
--- a/doc/html/footer.html
+++ b/doc/html/footer.html
@@ -67,7 +67,7 @@ href="http://www.splint.org/bugs.html">R
 Bugs</a> -
 <a
 href="http://www.splint.org/lists.html">Mailing
-Lists</a> &nbsp;&nbsp;<IMG src="http://sourceforge.net/sflogo.php?group_id=9538" width="0" height="0" alt="">
+Lists</a> 
 
 &nbsp;&nbsp;
 <a
